ARrehab/ARrehab/Tile.swift

```diff
@@ -13,11 +13,13 @@ import CoreGraphics
 class Tile : Entity, HasModel, HasCollision {
     
     //Class attributes
-    static let TILE_SIZE = SIMD3<Float>(0.5, 0.01, 0.5)
+    static let SCALE: Float = 1
+    static let TILE_SIZE = SIMD3<Float>(0.5 / SCALE, 0.01, 0.5 / SCALE)
```

Comment:
I believe scale is usually a multiplier instead of a divisor. Also why not scale depth as well. I know its already only 1 cm thick but might as well maintain symmetry.

Reply:
#22